fdbserver/TLogServer.actor.cpp

```diff
@@ -1758,7 +1759,17 @@ Future<Void> tLogPeekMessages(PromiseType replyPromise,
 
 	if (SERVER_KNOBS->ENABLE_VERSION_VECTOR && reqBegin > logData->persistentDataDurableVersion && !reqOnlySpilled &&
 	    reqTag.locality >= 0 && !reqReturnIfBlocked) {
-           wait(waitForMessagesForTag(logData, reqTag, reqBegin, SERVER_KNOBS->BLOCKING_PEEK_TIMEOUT));
+		state double startTime = now();
+		wait(waitForMessagesForTag(logData, reqTag, reqBegin, SERVER_KNOBS->BLOCKING_PEEK_TIMEOUT));
+		double latency = now() - startTime;
+		if (logData->blockingPeekLatencies.find(reqTag) == logData->blockingPeekLatencies.end()) {
+			UID ssID = nondeterministicRandom()->randomUniqueID();
```

Comment:
Is there a reason why you are choosing a random UID (instead of using something like LogData::logId)?

Reply:
I can make it `LogData::logId`